docs/getting-started.md

```diff
@@ -161,83 +194,147 @@ First we need to setup CLion
+        2. Then run `./tbots.py test [some_target_here] -- --run_sim_in_realtime`
+    2. For PyTests:
+        - With the visualizer: `./tbots.py test [some_target_here] -t`
+        - Without the visualizer: `./tbots.py test [some_target_here]`
+    3. For legacy C++ tests (#2581) without the visualizer:
+        - `./tbots.py test [some_target_here]`
+4. Run our SimulatedTacticTests in Thunderscope:
+    - This will launch the visualizer and simulate an AI Tactic on a single robot
+    1. For legacy C++ tests (#2581) with the visualizer:
+        - First, run Thunderscope configured for receiving protobufs over unix sockets correctly: `./tbots.py run thunderscope_main -- --visualize_cpp_test`
+        - Then run `./tbots.py test [some_target_here] -- --run_sim_in_realtime`
+    2. For PyTests:
+        - With the visualizer: `./tbots.py test [some_target_here] -t`
+        - Without the visualizer: `./tbots.py test [some_target_here]`
+    3. For legacy C++ tests (#2581) without the visualizer:
+        - `./tbots.py test [some_target_here]`
+</details>
+</details>
+
+</details>
 
 ## Debugging
+
+<details>
 Debugging from the command line is certainly possible, but debugging in a full IDE is *really* nice (plz trust us). 
 
 ### Debugging with CLion
 Debugging in CLion is as simple as running the above instructions for building CLion, but clicking the little green bug in the top right corner instead of the little green arrow!
 
 ### Debugging from the Command line
-To debug from the command line, first you need to build your target with the debugging flag - `bazel build -c dbg //some/target:here`. When the target builds, you should see a path `bazel-bin/<target>`. Copy that path, and run `gdb <path>`. Please see [here](https://www.cs.cmu.edu/~gilpin/tutorial/) for a tutorial on how to use `gdb` if you're not familiar with it. Alternatively, you could do `bazel run -c dbg --run_under="gdb" //some/target:here`, which will run the target in `gdb`. While this is taken directly from the bazel docs, gdb may sometimes hang when using `--run_under`, so building the target first with debugging flags and running afterwards is preferred.
-
+To debug from the command line, first you need to build your target with the debugging flag - `bazel build -c dbg //some/target:here`. When the target builds, you should see a path `bazel-bin/<target>`. Copy that path, and run `gdb <path>`. Please see [here](https://www.cs.cmu.edu/~gilpin/tutorial/) for a tutorial on how to use `gdb` if you're not familiar with it. Alternatively, you could do `bazel run -c dbg --run_under="gdb" //some/target:here`, which will run the target in `gdb`. While this is taken directly from the Bazel docs, gdb may sometimes hang when using `--run_under`, so building the target first with debugging flags and running afterwards is preferred.
+</details>
```

Comment:
what are these detail things

Reply:
<detail> Nima and I wanted to make the getting started guide more navigable instead of being a chunk of text so <detail> creates dropdowns that you'd optionally open up if it's relevant to you